src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,99 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.list.IntArrayStack;
+import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.list.ListInterface;
+
+public class DefaultCountingOutRhymer 
+// TODO what this does actually?
```

Comment:
It is explained in the beginning of the lab wiki. This is a library for "rhymers" (imaginary) game.